src/pdn/README.md

```diff
@@ -168,7 +167,7 @@ define_power_switch_cell
     -ground ground_pin
     -name name
     -power unswitched_power_pin
-    -power_switchable power_switchable_pin
```

Comment:
the tcl code has this as `power_switchable`

Reply:
Fixed. Thanks.